src/ray/common/ray_config_def.h

```diff
@@ -319,7 +319,7 @@ RAY_CONFIG(int64_t, max_resource_shapes_per_load_report, 100)
 RAY_CONFIG(bool, report_worker_backlog, true)
 
 /// The timeout for synchronous GCS requests in seconds.
-RAY_CONFIG(int64_t, gcs_server_request_timeout_seconds, 5)
+RAY_CONFIG(int64_t, gcs_server_request_timeout_seconds, 20)
```

Comment:
```suggestion
RAY_CONFIG(int64_t, gcs_server_request_timeout_seconds, 60)
```

RPC timeouts should be generous

Reply:
Set it to 60 seconds. Btw, isn't 30 seconds more reasonable as our heartbeat timeout is 30 seconds? (if there's 30s GCS server request delay, it is probably crashed by the heartbeat already?)

Reply:
It shouldn't have anything to do with that (orthogonal problem).